crmd/lrm.c

```diff
@@ -1971,8 +1971,10 @@ remote_node_init_status(const char *node_name, int call_opt)
 {
     int call_id = 0;
     xmlNode *update = create_xml_node(NULL, XML_CIB_TAG_STATUS);
+    xmlNode *state;
 
-    simple_remote_node_status(node_name, update,__FUNCTION__);
+    state = simple_remote_node_status(node_name, update,__FUNCTION__);
+    crm_xml_add(state, XML_NODE_IS_REMOTE_FENCED, "false");
```

Comment:
Suggestions:

- don't make the name of new attributes specific to remote nodes
  the concepts might be useful for normal nodes one day and it increases the fields to document for no real benefit

- set it to seconds since epoch

  That way you can then compare it to the time at which the monitor op failed to decide if the node needs (re-)fencing rather than rely on it being deleted at a specific time.